include/RAJA/policy/openmp_target/reduce.hpp

```diff
@@ -249,6 +249,18 @@ struct TargetReduce
   T finalVal;
 };
 
+template <typename T>
+struct DefaultLoc
```

Comment:
Why do we have this defined very similarly twice?

Reply:
One is a specialization which sets the default location to -1, and the other initializes with the constructor. It seems appropriate to keep both. Removing the default specialization template fails some tests in test-reduce.cpp.